playbooks/common/openshift-cluster/upgrades/pre/verify_health_checks.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Verify Host Requirements
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: "upgrade"
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
```

Comment:
If you're just configuring how the checks run per playbook, you can just add another variable (like `r_openshift_health_checker_playbook_context`) on the playbook and read that from the check. The only thing that's special about that context var is that it gets recorded in results so that the callback plugin can see it and adjust output.